packages/block-library/src/latest-posts/index.php

```diff
@@ -62,6 +62,10 @@ function render_block_core_latest_posts( $attributes ) {
 		$class .= ' columns-' . $attributes['columns'];
 	}
 
+	if ( isset( $attributes['displayPostDate'] ) && $attributes['displayPostDate'] ) {
```

Comment:
No, it isn't.

Since this is not JavaScript land, checking for `true === $attributes['displayPostDate']` or just `$attributes['displayPostDate']` will result in undefined index notices when the array key isn't set.

That's why the `isset()` check is needed and all other classes in that function are added this way.